geniza/corpus/migrations/0001_initial.py

```diff
@@ -21,6 +21,19 @@ class Migration(migrations.Migration):
             ],
             options={
                 'verbose_name_plural': 'Libraries',
+                'ordering': ['name']
             },
         ),
+        migrations.CreateModel(
+            name='LanguageScript',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('language', models.CharField(max_length=255)),
+                ('script', models.CharField(max_length=255)),
+                ('display_name', models.CharField(max_length=255, blank=True))
+            ],
+            options={
+                'ordering': ['language'],
+            },
+        )
```

Comment:
I didn't notice at first that you'd add this to the initial migration! It should probably be a separate migration (and I see now that I should probably have renamed the first one to make it more specific!).

I'd be open to combining all our initial models into a single migration as long as we all agree on that — in this case my db got into a weird state because the previous version of this migration was already applied.

Reply:
Oh good to know! I didn't know what the convention was. I'll make a new file